src/App.tsx

```
@@ -38,11 +47,235 @@ function Loading() {
+              <h1 className="text-lg font-semibold text-slate-900 dark:text-slate-100">
+                {siteTitle}
+              </h1>
+            </div>
+            <div className="flex items-center space-x-3">
+              <Breadcrumbs items={breadcrumbs} className="text-xs" />
+              <button
+                onClick={() => setMobileMenuOpen(!mobileMenuOpen)}
+                className="md:hidden p-2 text-slate-500 hover:text-slate-700 rounded-lg"
+                aria-label="Toggle navigation menu"
+              >
+                <svg className="w-6 h-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)